graylog2-server/src/main/java/org/graylog2/rest/resources/system/SessionsResource.java

```diff
@@ -120,6 +119,11 @@ public SessionResponse newSession(@Context ContainerRequestContext requestContex
         throw new NotAuthorizedException("Invalid username or password", "Basic realm=\"Graylog Server session\"");
     }
 
+    @GET
+    @ApiOperation(value = "Validate an existing session", notes = "Checks the session with the given ID: return OK if session is valid.")
```

Comment:
Just a bit of nit-picking: the description says it will return an OK if the session is valid, but it actually responds No Content. We could change the method to return an OK or just say it returns a success code.